src/target/riscv/riscv.c

```diff
@@ -4501,6 +4583,27 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.help = "Execute a sequence of 32-bit instructions using the program buffer. "
 			"The final ebreak instruction is added automatically, if needed."
 	},
+	{
+		.name = "equality_match_trigger_enable",
+		.handler = riscv_equality_match_trigger_enable,
```

Comment:
This name is already quite long, so if you like, I believe you can use just `eq` instead of the full word `equality`.

```suggestion
		.name = "eq_match_trigger_enable",
		.handler = riscv_eq_match_trigger_enable,
```

Reply:
Addressed